include/ctranslate2/storage_view.h

```diff
@@ -238,6 +240,36 @@ namespace ctranslate2 {
 
     friend std::ostream& operator<<(std::ostream& os, const StorageView& storage);
 
+    template <typename T>
+    void print_tensor(std::ostream& os, const T* data, const std::vector<dim_t>& shape, size_t dim, size_t offset, int indent) const {
```

Comment:
Can you move it to storage_view.cc then keep the location of ``PRINT_MAX_VALUES``like before